doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,255 @@
+      This builds the project.
+
+   You can find the output of the build, which includes the merged HEX file containing both the application and the SPM, in the :file:`zephyr` subfolder in the build directery.
+
+#. To program the sample or application:
+
+   a. Connect the Thingy:53 to the debug out port on a 10-pin external debug probe, for example, nRF5340 DK (Development Kit), using a 10-pin JTAG cable.
+   #. Connect the external debug probe to the PC using a USB cable.
+   #. Make sure that the Thingy:91 and the external debug probe are powered on.
+   #. in SES, select :guilabel:`Target` > :guilabel:`Connect J-Link`.
+   #. Select guilabel:`Target` > :guilabel:`Download zephyr/merged.hex` to program the sample or application onto Thingy:53.
+   #. The device will reset and run the programmed sample or application.
+
+Building and programming using Visual Studio Code
+*************************************************
+
+Complete the following steps after installing the `nRF Connect for VS Code`_ extension in Microsoft Visual Studio Code:
+
+1. Open VS code.
+#. Click on the :guilabel:`nRF Connect for VS Code` icon in the left navigation bar and then click :guilabel:`Open`.
+
+   .. figure:: images/ug_thingy53_vscode_welcome.png
+      :alt: nRF Connect for VS Code welcome page
+      :width: 80%
+
+      The welcome page for nRF Connect for VS Code.
+
+#. Click on :guilabel:`Add an existing applicaton to workspace...`.
+#. In the prompt, navigate to the folder containing the sample you want to build, such as :file:`nrf\\samples\\bluetooth\\peripheral_lbs`.
+   You should now see the selected application in the :guilabel:`Applications` window in the lower left corner.
+
+   .. note::
+
+     The sample folder must contain a :file:`prj.conf` file.
+
+   .. figure:: images/ug_thingy53_vscode_appwindow.png
+      :alt: nRF Connect for VS Code, application shown in Application window
+      :width: 80%
+
+      The peripheral_lbs application can now be seen in the :guilabel:`Application` window.
+
+#. Click on the :guilabel:`Add Build Configuration` button in the :guilabel:`Applications` window, or click on the text stating :guilabel:`No build configurations. Click to create one`.
+   This will open the :guilabel:`Generate Configuration` window in a new tab.
+#. Select the thingy53_nrf5340_cpuapp as the target board and click :guilabel:`Generate Config`, which generates the config file and triggers the build process.
+
+   .. figure:: images/ug_thingy53_vscode_genconfig1.png
+      :alt: nRF Connect for VS Code, Generate Configuration window
+      :width: 80%
+
+      The :guilabel:`Generate Configuration` window, with thingy53_nrf5340_cpuapp as the board.
+
+#. Once the build configuration and the build are complete, an :guilabel:`Actions` window appears in the lower left corner of VS Code.
+   In this window you can trigger the build process, flash the built sample and start a debug session.
```

Comment:
```suggestion
#. When the build configuration and the build are complete, an :guilabel:`Actions` window appears in the lower left corner of VS Code.
   In this window, you can trigger the build process, program the built sample and start a debug session.
```

Reply:
Is this a step?

Reply:
I understood this to be explanation of options, not a compulsory step. I changed the "and" to "or" to make it more clear.